jaxrpc-ri/pom.xml

```diff
@@ -13,90 +13,182 @@
     <url>https://github.com/eclipse-ee4j/jax-rpc-ri</url>
-    
+
     <scm>
         <connection>scm:git:git@github.com:eclipse-ee4j/jax-rpc-ri.git</connection>
         <developerConnection>scm:git:git@github.com:eclipse-ee4j/jax-rpc-ri.git</developerConnection>
         <url>https://github.com/eclipse-ee4j/jax-rpc-ri</url>
     </scm>
-    
+
     <licenses>
         <license>
             <name>EPL-2.0</name>
             <url>http://www.eclipse.org/legal/epl-2.0</url>
         </license>
     </licenses>
-    
-    <dependencies>
-        <dependency>
-            <groupId>javax.mail</groupId>
-            <artifactId>javax.mail-api</artifactId>
-            <version>1.6.2</version>
-        </dependency>
-        <dependency>
-            <groupId>javax.activation</groupId>
-            <artifactId>activation</artifactId>
-            <version>1.1.1</version>
-        </dependency>
-        <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
-            <version>4.0.1</version>
-        </dependency>
-        <dependency>
-            <groupId>javax.xml.rpc</groupId>
-            <artifactId>javax.xml.rpc-api</artifactId>
-            <version>1.1.2</version>
-        </dependency>
-        <dependency>
-            <groupId>javax.xml.soap</groupId>
-            <artifactId>saaj-api</artifactId>
-            <version>1.3.5</version>
-        </dependency>
-        <dependency>
-            <groupId>javax.xml.soap</groupId>
-            <artifactId>saaj-impl</artifactId>
-            <version>1.3</version>
-        </dependency>
-        <dependency>
-            <groupId>com.sun.xml</groupId>
-            <artifactId>relaxngDatatype</artifactId>
-            <version>1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.sun.xml.fastinfoset</groupId>
-            <artifactId>FastInfoset</artifactId>
-            <version>1.2.15</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.ant</groupId>
-            <artifactId>ant</artifactId>
-            <version>1.9.13</version>
-        </dependency>
 
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-            <scope>test</scope>
-        </dependency>
-    </dependencies>
+    <organization>
+        <name>Eclipse Foundation</name>
+        <url>https://www.eclipse.org</url>
+    </organization>
+
+    <properties>
+        <mail.api.version>1.6.3</mail.api.version>
+        <activation.api.version>1.2.1</activation.api.version>
+        <servlet.api.version>4.0.2</servlet.api.version>
+        <rpc.api.version>1.1.3</rpc.api.version>
+        <soap.api.version>1.4.1</soap.api.version>
+        <saaj.version>1.5.1</saaj.version>
+        <xerces.version>2.6.2-jaxb-1.0.6</xerces.version>
+        <fastInfoset.version>1.2.15</fastInfoset.version>
```

Comment:
1.2.16 is the jakarta provided version - check with @Tomas-Kraus to see it's ready